crates/rolldown/src/stages/generate_stage/compute_cross_chunk_links.rs

```diff
@@ -1,22 +1,26 @@
+use std::hash::BuildHasherDefault;
 use std::{borrow::Cow, sync::Mutex};
 
-use crate::{chunk_graph::ChunkGraph, utils::is_in_rust_test_mode};
-
 use super::GenerateStage;
+use crate::{chunk_graph::ChunkGraph, utils::is_in_rust_test_mode};
 use index_vec::{index_vec, IndexVec};
+use indexmap::IndexSet;
 use rolldown_common::{
   ChunkId, ChunkKind, CrossChunkImportItem, ExportsKind, ExternalModuleId, ImportKind, ModuleId,
   NamedImport, OutputFormat, SymbolRef,
 };
 use rolldown_rstr::{Rstr, ToRstr};
-use rolldown_utils::rayon::{IntoParallelIterator, ParallelBridge, ParallelIterator};
-use rustc_hash::{FxHashMap, FxHashSet};
+use rolldown_utils::rayon::IntoParallelIterator;
+use rolldown_utils::rayon::{ParallelBridge, ParallelIterator};
+use rustc_hash::{FxHashMap, FxHashSet, FxHasher};
 
 type ChunkMetaImports = IndexVec<ChunkId, FxHashSet<SymbolRef>>;
 type ChunkMetaImportsForExternalModules =
   IndexVec<ChunkId, FxHashMap<ExternalModuleId, Vec<NamedImport>>>;
 type ChunkMetaExports = IndexVec<ChunkId, FxHashSet<SymbolRef>>;
-type IndexCrossChunkImports = IndexVec<ChunkId, FxHashSet<ChunkId>>;
+type IndexCrossChunkImports = IndexVec<ChunkId, Vec<ChunkId>>;
```

Comment:
Let me try to make things clear again:

Do not change `IndexCrossChunkImports` from `IndexVec<ChunkId, FxHashSet<ChunkId>>` to `IndexVec<ChunkId, Vec<ChunkId>>`.

- `Vec<ChunkId>` may contains duplicated `ChunkId`
- Don't use `IndexSet` on `IndexCrossChunkImports`. https://github.com/rolldown/rolldown/pull/1002/files#r1583073224

Reply:
I missing revert it.